mock/mock.go

```diff
@@ -501,7 +526,7 @@ type Arguments []interface{}
 const (
 	// Anything is used in Diff and Assert when the argument being tested
 	// shouldn't be taken into consideration.
-	Anything string = "mock.Anything"
+	Anything = "mock.Anything"
```

Comment:
FYI, for future contributions, it would be best to keep this kind of things in a different PR.